src/components/App/AppContainer.tsx

```diff
@@ -29,9 +43,17 @@ const mergeProps = (stateProps: any, dispatchProps: any, ownProps: any) => {
     dispatchProps.customerProps,
     ownProps.customerProps
   );
+  const orderProps = Object.assign(
+    {},
+    stateProps.orderProps,
+    dispatchProps.orderProps,
+    ownProps.orderProps
+  );
 
   return {
```

Comment:
The way this state is being passed to the App component is a problem. I need your help to find a better way